.github/workflows/branch.yaml

```diff
@@ -51,6 +51,6 @@ jobs:
       - name: Run bandit
         run: poetry run bandit .
       - name: Run safety
-        run: poetry run safety check
+        run: poetry run safety check -i 62044,67599,67895
```

Comment:
I think running `poetry update` will remove all the safety errors but 67599. Not 100 % sure.